automation/bake/Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyri
 
 RUN apt-get update && apt-get install -y google-cloud-sdk
 
+RUN apt-get update && apt-get install -y kubectl
```

Comment:
This should move to dockerfiles/Dockerfile-coda-daemon if it's going to be required for health checks